R/coops.R

```diff
@@ -34,17 +34,22 @@
 #'  measure of atmospheric clarity
 #'  \item humidity - Relative humidity as measured at the station
 #'  \item salinity - Salinity and specific gravity data for the station
+#'  \item one_minute_water_level - One minute water level data for the station
+#'  \item predictions - 6 minute predictions water level data for the station
 #'  \item hourly_height - Verified hourly height water level data for
 #'  the station
 #'  \item high_low - Verified high/low water level data for the station
 #'  \item daily_mean - Verified daily mean water level data for the station
 #'  \item monthly_mean - Verified monthly mean water level data for the station
-#'  \item one_minute_water_level - One minute water level data for the station
-#'  \item predictions - 6 minute predictions water level data for the station
 #'  \item datums - datums data for the stations
 #'  \item currents - Currents data for currents stations
 #' }
 #'
+#' Maximum Durations in a Single Call:
+#' Products water_level through predictions allow requests for up to 31 days of data.
```

Comment:
for these lines 49-51, don't go over 80 character width, so jus wrap to next line if needed.

and put them in a list like 

```r
#' \itemize{
#'  \item xxx
#'  \item xxx
#'  \item xxx
#' }
```